client_plugin/utils/refcount/refcnt.go

```diff
@@ -387,7 +380,7 @@ func (r *RefCountsMap) discoverAndSync(c *client.Client, d drivers.VolumeDriver)
 		log.Debugf("  Mounts for %v", ct.Names)
 		for _, mount := range containerJSONInfo.Mounts {
 			// check if the mount location belongs to vmdk plugin
-			if isVMDKMount(mount.Source) != true {
+			if isVMDKMount(mount.Driver) != true {
```

Comment:
Good point and no. But it could be documented to use a _recommended_ alias - something that has  _vsphere_ in it. I mean if they install the vsphere plugin and call it by another org's name that won't be right. Plus playing with alias names can mean that volumes won't be accessible across aliases though they are managed by the same plugin. What do you suggest.